docs/builds/guides/frameworks/react.md

```diff
@@ -114,19 +173,26 @@ If you use the {@link framework/guides/document-editor Document editor}, you nee
 import DecoupledEditor from '@ckeditor/ckeditor5-build-decoupled-document';
 
 class App extends Component {
+	editor = null;
+
 	render() {
 		return (
 			<div className="App">
 				<h2>CKEditor 5 using a custom build - DecoupledEditor</h2>
 				<CKEditor
-					onInit={ editor => {
+					onReady={ editor => {
 						console.log( 'Editor is ready to use!', editor );
 
 						// Insert the toolbar before the editable area.
 						editor.ui.getEditableElement().parentElement.insertBefore(
 							editor.ui.view.toolbar.element,
 							editor.ui.getEditableElement()
 						);
+
+						this.editor = editor;
+					} }
+					onError={ () => {
+						this.editor.ui.view.toolbar.element.remove();
```

Comment:
We shouldn't remove the toolbar if the editor will not restart. Also, would be good to write a dedicated section about the decoupled editor and the integration with the latest version (3+).

Reply:
> We shouldn't remove the toolbar if the editor will not restart.

If the editor won't restart, then the missing toolbar actually won't change anything ;)

Reply:
Right now (version 2.x), if the editor crashes, the toolbar won't disappear.

Reply:
But the editor restarts and the correct toolbar will be recreated, so where's the problem? In the previous version, the editor would stay crashed.

Reply:
I can add an if statement though.